packages/itwinui-react/src/core/Table/Table.test.tsx

```diff
@@ -3287,81 +2600,25 @@ it('should resize last and closest column on the left when table width would dec
       id: 'view',
       Header: 'view',
       Cell: () => <>View</>,
-    },
-  ];
-  const { container } = renderComponent({
-    columns,
-    isResizable: true,
-    columnResizeMode: 'expand',
-  });
-
-  // Initial render
-  triggerResize({ width: 300 } as DOMRectReadOnly);
-
-  const rows = container.querySelectorAll('.iui-table-body .iui-table-row');
-  expect(rows.length).toBe(3);
-
-  const resizers = container.querySelectorAll('.iui-table-resizer');
-  expect(resizers.length).toBe(3);
-
-  // Resize past table width
-  fireEvent.mouseDown(resizers[2], { clientX: 300 });
-  fireEvent.mouseMove(resizers[2], { clientX: 250 });
-  // fireEvent.mouseMove(resizers[2], { clientX: 50 });
-  fireEvent.mouseUp(resizers[2]);
-
-  const headerCells = container.querySelectorAll<HTMLDivElement>(
-    '.iui-table-header .iui-table-cell',
-  );
-  expect(headerCells).toHaveLength(3);
-
-  // Headers widths sum are not lower than table width
-  expect(headerCells[0].style.width).toBe('100px');
-  expect(headerCells[1].style.width).toBe('150px');
-  expect(headerCells[2].style.width).toBe('50px');
-});
-
-it('should not show resizer when column has disabled resizing when resize mode is expand', () => {
```

Comment:
If we remove the `getBoundingClientRect` mock, can this test be brought back?

Reply:
I think it's already covered once? The columnResizeMode has no effect on it.

![image](https://github.com/iTwin/iTwinUI/assets/9084735/3fe400ea-d4a6-4320-a455-05e58a3f2903)